### PR TITLE
[TRAFODION-2482] documentation for python installer

### PR DESCRIPTION
**Pull request by mkby**

### CODE REVIEW

File: docs/provisioning_guide/src/asciidoc/_chapters/introduction.adoc [hunk 1]
```diff
@@ -165,24 +165,23 @@ Next, you unpack the tar file.
...
 $ cd $HOME/trafodion-downloads
-$ tar -zxf apache-trafodion-installer-1.3.0-incubating-bin.tar.gz -C $HOME/trafodion-installer
-$ 
+$ tar -zxf apache-trafodion-pyinstaller-2.1.0-incubating.tar.gz -C $HOME/trafodion-installer
+$
 ```
 
 <<<
-The {project-name} Installer supports two different modes:
+The {project-name} Installer supports three different modes:
```
Comment (svarnau) [inline docs/provisioning_guide/src/asciidoc/_chapters/introduction.adoc]: where is the third mode described?

File: docs/provisioning_guide/src/asciidoc/_chapters/quickstart.adoc [hunk 2]
```diff
@@ -48,60 +48,55 @@ NOTE: You can download and install the {project-name} Clients once you've instal
...
-```
-$ # Download the Trafodion Server binaries
-$ wget http://apache.cs.utah.edu/incubator/trafodion/trafodion-1.3.0.incubating/apache-trafodion-1.3.0-incubating-bin.tar.gz
+$ wget http://apache.cs.utah.edu/incubator/trafodion/trafodion-2.1.0.incubating/apache-trafodion_server-2.1.0-RH-x86_64-incubating.tar.gz
 Resolving http://apache.cs.utah.edu... 192.168.1.56
 Connecting to http://apache.cs.utah.edu|192.168.1.56|:80... connected.
 HTTP request sent, awaiting response... 200 OK
 Length: 214508243 (205M) [application/x-gzip]
-Saving to: "apache-trafodion-1.3.0-incubating-bin.tar.gz"
+Saving to: "apache-trafodion_server-2.1.0-RH-x86_64-incubating.tar.gz"
```
Comment (svarnau) [inline docs/provisioning_guide/src/asciidoc/_chapters/quickstart.adoc]: should that be RH6??

Comment (mkby) [inline docs/provisioning_guide/src/asciidoc/_chapters/quickstart.adoc]: This is the package name after `make package` on my system, is there anything wrong with my environment?

**Comment by sandhyasun** [comment]
This seems like it's ready for merging. @svarnau were you concerened about the RH in the package name ? Do you want to ccreate a separate JIRA to look into the package name ?

**Comment by svarnau** [comment]
The sqf/Makefile does:
OS_TYPE = RH
OS_MAJOR ?=  $(shell lsb_release -rs | cut -f1 -d.)
PKG_TYPE="server-$(TRAFODION_VER)-${OS_TYPE}${OS_MAJOR}-${P_TYPE}-incubating"

I guess if the build system does not have lsb_release command installed, it might come out blank instead of "6".  But the intent is that the name have RH6 or RH7, not just RH.

**Comment by sandhyasun** [comment]
I see the same issue when I build on a VM. So is there a new RPM we need to add tot he list of needed rpms before a build ?

**Comment by sandhyasun** [comment]
Actually I see lsb release is present on the VM but the output of the package is still RH -
[centos@sandhya-amb-2 distribution]$ lsb_release -rs
6.7
[centos@sandhya-amb-2 distribution]$ lsb_release -rs | cut -f1 -d.
6

--
These are the package names : 
apache-trafodion_clients-2.1.0-RH-x86_64-debug.tar.gz
apache-trafodion_installer-2.1.0-incubating.tar.gz
apache-trafodion_pyinstaller-2.1.0-incubating.tar.gz
apache-trafodion-regress.tgz
apache-trafodion_server-2.1.0-RH-x86_64-debug.tar.gz

**Comment by svarnau** [comment]
Interesting. OS_MAJOR is being set with ?=, which means it could be inheriting it from an environment variable.  Otherwise, I don't understand why OS_MAJOR is not being set for some people.  It works in my dev environment and in jenkins.

**Comment by sandhyasun** [comment]
My fresh build on a dev workstation did have the correct RH6 package name. It's only on a openstack VM I saw this. So the setting of OS_MAJOR seems to not be deterministic.

**Comment by sandhyasun** [comment]
Eitherway, we shoudl probably get this documentation change merged  in ?

**Comment by svarnau** [comment]
Agreed, this is a tangent and a minor issue for the doc.

**Pull request merged**